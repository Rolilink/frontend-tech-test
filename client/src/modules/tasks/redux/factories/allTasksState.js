import httpFactory from './http';
import allTasksFactory from './allTasks';

export default () => ({
  ...httpFactory(),
  ...allTasksFactory(),
});
