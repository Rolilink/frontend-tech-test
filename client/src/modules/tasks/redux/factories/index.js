import allTasks from './allTasks';
import http from './http';
import state from './allTasksState';

export {
  allTasks,
  http,
  state,
};
