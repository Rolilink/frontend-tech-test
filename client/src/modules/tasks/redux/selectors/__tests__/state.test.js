import getState from '../state';
import { allTasks as allTasksFactories, http as httpFactory } from '../../factories';

const state = {
  tasks: {
    allTasks: allTasksFactories.state(),
    createTask: httpFactory(),
  },
};

describe('Tasks.Redux.Selectors.getState', () => {
  it('should get tasks state from store state', () => {
    expect(getState(state)).toMatchSnapshot();
  });
});
