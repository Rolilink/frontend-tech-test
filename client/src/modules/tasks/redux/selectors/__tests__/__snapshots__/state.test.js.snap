// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tasks.Redux.Selectors.getState should get tasks state from store state 1`] = `
Object {
  "allTasks": Object {
    "error": null,
    "isFetching": false,
    "tasks": Object {},
  },
  "createTask": Object {
    "error": null,
    "isFetching": false,
  },
}
`;
